ci/travis/install-mssql-pdo_sqlsrv.sh

```diff
@@ -4,4 +4,4 @@ set -ex
 
 echo "Installing extension"
 
-pecl install pdo_sqlsrv-5.7.0preview
+pecl install pdo_sqlsrv-5.8.1
```

Comment:
This looks like a good change, but does it related to your changes? Shouldn't it be committed separately?